app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
```

Comment:
- [ ] According to the review guidelines, your design for the **login** and **registration** forms should match those provided to you in the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) as seen in the screenshot below:
![image](https://github.com/mpa-mxiang/Blog-app/assets/45460192/d4d95d89-d7c2-4bb3-8d86-1315e0fbc81c)
### Your Design
![image](https://github.com/mpa-mxiang/Blog-app/assets/45460192/bee1ea4f-5542-40b9-bee2-442876d248df)
### Required
![image](https://github.com/mpa-mxiang/Blog-app/assets/45460192/4bd39bad-a1da-47a9-b257-81f35184009c)